src/lib/layouts/bengali.js

```diff
@@ -0,0 +1,18 @@
+const bengali = {
```

Comment:
Hey @adnan360!

It appears your proposed layout differs from other virtual keyboards in Bengali:

https://gate2home.com/Bengali-Keyboard
https://vkboard.com/virtual-bangla-keyboard/
https://www.branah.com/bangla

Could you update your PR or cite your references?

Thank you,
Francisco Hodge

Reply:
Sorry, I can't cite anything. I actually created this layout myself. :)

I designed the layout so that it's comfortable to click-type and most used letters/characters are put in the non-shift state. This should be comfortable to use since it will be used with mouse pointer or touch. Feel free to ask any Bangla speakers to use it and I'm open to suggestions. I used it myself and I liked it as an on screen keyboard layout.

I was motivated to make my own because there has been copyright claims on layouts which are based on or similar to Bijoy layout by it's maker Ananda Computers/Mustafa Jabbar. For example, if you visit [ridmik keyboard's website](https://ridmik.com/) and expand "Why there is no "Unijoy" layout?" it says:
> "Unijoy" was removed because of a copyright dispute. Although we do not believe that it was an accurate claim.

The [Unijoy](https://en.wikipedia.org/wiki/Bengali_input_methods#Unijoy_keyboard) layout is actually made by a FOSS friendly community of devs and is similar to [Bijoy](https://en.wikipedia.org/wiki/Bengali_input_methods#Bijoy_2). They were attacked my Mustafa Jabbar for violating copyright. The unijoy devs had an anger filled discussion with Mustafa Jabbar and somehow they allowed themselves to use it. Their [website](https://www.ekushey.org/?page/uni_joy_layout) says:
> Note: Apparently Mustafa Jabbar has attempted to claim the Bijoy keyboard layout as his personal property. We do not accept that a proprietorship approach is appropriate to a public standard.
> This software was produced by altruistic volunteers to promote Bangla usage by increasing the ease with which Unicode character data can be input.

But it seems if anyone else uses Unijoy as their layout, they attack with copyright claim (as is the case for Ridmik).

This layout that I made is available under MIT license (same as this repo). I can license more freely (CC0) if needed. I can help with implementing the layouts you mentioned because at least they're not Bijoy based. But I'm not sure if they will be ok if we use it in this project. Maybe we'll have to contact the authors to be sure. I think a safer approach would be to use mine instead, since I already announced it as MIT licensed.

Reply:
Sorry for the delay, I appreciate the explanation - I plan to resolve/merge the PR this weekend (just so I have the time to update the docs as well). Thanks!